quickwit/quickwit-metastore/migrations/postgresql/16_create-index-split-uid.down.sql

```diff
@@ -0,0 +1 @@
+DROP INDEX IF EXISTS index_splits_index_uid;
```

Comment:
```suggestion
DROP INDEX IF EXISTS splits_index_uid_idx;
```

Reply:
The naming convention for pg constraints/indexes is`{tablename}_{columnname(s)}_{suffix}` where suffix is `pkey`, `fkey`. `idx`, etc.